### PR TITLE
Add TV shorturl

### PR DESCRIPTION
**Pull request by tahabi**
I messed up in git on the previous PR this one should fix that

### CODE REVIEW

File: modules/ocf_www/manifests/site/shorturl.pp [hunk 1]
```diff
@@ -79,6 +79,7 @@
       {rewrite_rule => '^/stf-cost-breakdown$ https://docs.google.com/spreadsheets/d/1U3YfU5S1hyi4c9u1vME84lz5hQPz23LKWCaD8mq4JHI/edit [R]'},
       {rewrite_rule => '^/tw(/.*)?$ https://ocf.io/twitter$1 [R]'},
       {rewrite_rule => '^/twitter(/.*)?$ https://twitter.com/ucbocf$1 [R]'},
+      {rewrite_rule => '^/tv(/.*)?$ https://ocf.berkeley.edu/tv$1 [R]'},
```
Comment (chriskuehl) [inline modules/ocf_www/manifests/site/shorturl.pp]: should be www.ocf.b.e

**Comment by matthew-mcallister** [comment]
I invite you to squash all these commits into one.

**Comment by tahabi** [comment]
yeah I messed up again

**Comment by matthew-mcallister** [comment]
Protip: `git commit --amend`.

**Comment by tahabi** [comment]
this was a more severe error than that, I'm just going to delete everything and restart. I tried git rebase -i fixup and that created some duplicate commits.

**Comment by matthew-mcallister** [comment]
If you use `git commit --amend`, you won't have to rebase.

**Comment by tahabi** [comment]
fixed it

**Pull request merged**